### PR TITLE
added functions key_to_table_key & key_to_context_key to common server python

### PR DESCRIPTION
**Pull request by yardensade**
Do not merge until review, thanks.

### CODE REVIEW

File: Scripts/script-CommonServerPython.yml [hunk 1]
```diff
@@ -1079,6 +1079,39 @@ script: |-
...
+
+        :return: The converted string
+        :rtype: ``string``
+      """
+      if isinstance(string, basestring):
+          return " ".join(word.capitalize() for word in string.replace("_"," ").split())
+      else:
+          return_error('The key is not a string: {}'.format(string))
+
+  def key_to_context_key(string):
```
Comment (hirshbergshachar) [inline Scripts/script-CommonServerPython.yml]: I recommend to rename it to `key_to_upper_camelcase`

Comment (yardensade) [reply Scripts/script-CommonServerPython.yml]: Technically its more like key_to_title except the spacing part.

File: Scripts/script-CommonServerPython.yml [hunk 2]
```diff
@@ -1079,6 +1079,39 @@ script: |-
...
+        Checks if string, change underscores to spaces, capitalize every word.
+        Example: "one_two" to "One Two"
+
+        :type string: ``str``
+        :param string: The string to be converted (required)
+
+        :return: The converted string
+        :rtype: ``string``
+      """
+      if isinstance(string, basestring):
```
[the remark this answers is not in the document]
Comment (yaakovi) [reply Scripts/script-CommonServerPython.yml]: the common needs to be compatible with both versions

[the remark this answers is not in the document]
Comment (yardensade) [reply Scripts/script-CommonServerPython.yml]: Good to know :)

File: Scripts/script-CommonServerPython.yml [hunk 3]
```diff
@@ -1079,6 +1079,39 @@ script: |-
...
+       Example: "one_two" to "OneTwo"
+
+       :type string: ``str``
+       :param string: The string to be converted (required)
+
+       :return: The converted string
+       :rtype: ``string``
+      """
+      if isinstance(string, basestring):
+          return "".join(word.capitalize() for word in string.split('_'))
```
[the remark this answers is not in the document]
Comment (yardensade) [reply Scripts/script-CommonServerPython.yml]: But we split on _ not on any type of space, we only return add the space ourselves.

**Pull request merged**